src/data/markdown/docs/02 javascript api/07 k6-experimental/04 tracing.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "tracing"
+excerpt: "k6 Tracing experimental API"
+---
+
+<ExperimentalBlockquote />
+
+
+This experimental module instruments k6 scripts HTTP calls with tracing information. 
+
+
+| Class/Function                                                           | Description                                                                                                               |
+| :----------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------ |
+| [instrumentHTTP](/javascript-api/k6-experimental/tracing/instrumenthttp) | instruments the k6 http module with tracing capabilities.                                                                 |
+| [Client](/javascript-api/k6-experimental/tracing/client)                 | configurable Client that exposes instrumented HTTP operations and allows selectively instrumenting requests with tracing. |
+| [Options](/javascript-api/k6-experimental/tracing/options)               | Options allows to configure the tracing instrumentation behavior.                                                         |
+
+## Example
+
+This example demonstrates how to use the tracing API to instrument every HTTP request made in a script with tracing information.
+
+<CodeGroup labels={["example-tracing-module.js"]} lineNumbers={[]} showCopyButton={[true]}>
+
+```javascript
+import http from "k6/http";
+import { check } from "k6";
+import tracing from "k6/experimental/tracing";
+
+// instrumentHTTP will ensure that all requests made by the http module
+// from this point forward will have a trace context attached.
+//
+// The first argument is a configuration object that
+// can be used to configure the tracer.
+//
+// Currently supported HTTP methods are: get, post, put, patch, head,
+// del, options, and request.
+tracing.instrumentHTTP({
+propagator: "w3c",
```

Comment:
```suggestion
  propagator: "w3c",
```

Also, maybe add a `// possible values: "w3c", "jaeger"`? This is mentioned and explained on other pages, but since this is the first example users will see, just to give a heads-up for what's possible.